handbook/engineering/web/index.md

```diff
@@ -1,22 +1,83 @@
-<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 35%; float: right; margin-left: 1rem">
+
 
 # Web team
 
-The web team owns the maintenance and expansion of our web application and code host integrations as vehicles to deliver the value of [extensions](https://docs.sourcegraph.com/extensions) to our users.
+<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 120px; float: right; padding: 0 1rem">
+
+The Web team owns the maintenance and expansion of our web application and code host integrations as vehicles to deliver the value of [extensions](https://docs.sourcegraph.com/extensions) to our users.
+We also define and maintain the standards and tooling for how we do web development at Sourcegraph.
+<div style="clear: both"></div>
 
-This is a large ownership area, so the team creates a focused plan each iteration, by agreeing on an appropriately small set of [iteration goals](../../../company/goals/index.md). Each goal should have more than one teammate working on it.
+## Members
 
-<div style="clear: both"/>
+- [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) ([Product Manager](../../product/roles/product_manager.md))
+- [Jean du Plessis](../../../company/team/index.md#jean-du-plessis-he-him) ([Engineering Manager](../roles.md#engineering-manager))
+  - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
+  - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
+  - [T.R.](../../../company/team/index.md##full-name-optional-pronouns) _(Starts 2021-01-11)_
+  
+ℹ️ _Joel is the acting PM for this team until we are able to hire a dedicated PM for it. Once that happens, he will focus entirely on [Code Insights](../code-insights/index.md)._  
 
 ## Contact
 
-- #web channel or @web-team in Slack.
+- [#web](https://sourcegraph.slack.com/archives/CMT39K56Z) channel or @web-team in Slack.
 - [team/web](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/web) label and [@sourcegraph/web](https://github.com/orgs/sourcegraph/teams/web) team on GitHub.
 
 ## [Goals](goals.md)
 
 See [goals](goals.md)
 
+## Growth plan
+
+ℹ️ _Updated TODO_
+
+We are actively hiring [frontend engineers](../hiring/software-engineer-frontend.md) to grow the Web team. 
+When this team team gets big enough, we expect to split into two teams **Frontend platform and code browsing** and **Extensions and Integrations**
```

Comment:
```suggestion
When this team team gets big enough, we expect to split into two teams **Frontend platform and code browsing** and **Extensions and Integrations**. We will only make this split once we have identified a dedicated engineering manager and a dedicated product manager.
```
I would combine the note about "only splitting when we have an EM/PM" with the note up here and delete it below.